src/search/search-executor/search-enrichers.ts

```diff
@@ -54,6 +54,9 @@ export const Enrichments = {
 				const resolved = resolveByName(info.name, info.environment ?? initializeCleanEnvironments());
 				if(resolved) {
 					content.targets.push(...resolved.map(r => r.name).filter(i => i !== undefined));
+				} else {
+					// if we can't resolve the name to actual memory, we still want to show it as a vague target
```

Comment:
maybe issue a warning in these cases?

Reply:
doesn't this happen whenever a library function is used in general, since we don't analyze those at the moment?

Reply:
a warning for us not for the users of the linter, because this may also relate to an error in the search